.github/tests/namespace-override/.env

```diff
@@ -0,0 +1 @@
+export EXTRA_HELM_ARGS="--values=examples/production/values.yaml"
```

Comment:
This should not be part of this test. I would expect just the namespace override.